### PR TITLE
Allow initiative tracking to go backwards

### LINKED ISSUE #60: Allow initiative tracking to go backwards
I want to be able to go back in initiative order in the case that I click 'end turn' by mistake or realize there was more that could be done on a previous turn.

### PR DESCRIPTION
**Pull request by AndrewD713**
Closes #60

### CODE REVIEW

**Review by morkusporkus — APPROVED** [review]

File: src/Maui/Features/Combat/Manage.razor.cs [hunk 1]
```diff
@@ -184,7 +198,7 @@ public void EndCombatEncounter()
 
         public bool IsCreaturesTurn(InitiatedCreature creature) => _currentTurnCreature == creature;
 
-        public bool IsCombatActive() => _currentTurnCreature is not null;
+        public bool IsCombatActive() => _initiatedCreatures.Any();
```
Comment (benjaminsampica) [inline src/Maui/Features/Combat/Manage.razor.cs:201]: I dont think combat is active if the list has things in it. That’ll probably make sense if Add() wasn’t populating it.

Maybe Add() shouldn’t populate that list, instead keeping the creatures until combat is started?

File: src/Maui/Features/Combat/Manage.razor [hunk 2]
```diff
@@ -8,7 +8,8 @@
     </MudItem>
     <MudItem xs="12">
         <MudButton Variant="Variant.Outlined" Color="Color.Secondary" OnClick="OnBeginCombatClicked">Begin Combat</MudButton>
-        <MudButton Variant="Variant.Outlined" Color="Color.Tertiary" OnClick="OnEndTurnClicked" Disabled=@(!_combatEncounter.IsCombatActive())>End Turn</MudButton>
+        <MudButton Variant="Variant.Outlined" Color="Color.Tertiary" OnClick="OnPreviousTurnClicked" Disabled=@(!_combatEncounter.IsCombatActive())>Previous Turn</MudButton>
```
Comment (benjaminsampica) [inline src/Maui/Features/Combat/Manage.razor:11]: Now that there’s two, maybe a button group with three buttons might look nice? Turn could be a text variant? Doesn’t have to be a button grp either.

< Prev | Turn | Next >

Thoughts?

File: src/Maui/Features/Combat/Manage.razor.cs (hunk 1 above)
Comment (AndrewD713) [reply src/Maui/Features/Combat/Manage.razor.cs:201]: See PR #70

File: src/Maui/Features/Combat/Manage.razor (hunk 2 above)
Comment (AndrewD713) [reply src/Maui/Features/Combat/Manage.razor:11]: See PR #70

**Pull request merged**